app/Http/Controllers/PollController.php

```diff
@@ -19,7 +26,10 @@ public function create(Request $request): JsonResponse
         $poll->name = $name;
         $poll->description = $description;
         $poll->end_date = $endDate;
-        $poll->phase = $phase;
+
+        if($phase!=null){
```

Comment:
USE SPACES!!!!!

```suggestion
        if ($phase != null) {
```
alternative:
```suggestion
        if ($phase) {
```